<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>droneui</class>
 <widget class="QWidget" name="droneui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>droneui</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="taskMenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Task1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Task2</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="executeButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>310</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Calistir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Takeoff Duration</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Formation Type</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="parameter2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wait Duration</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="parameter3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Landing Duration</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="parameter4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="parameter1">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Konuma Git</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
